src/foam/u2/crunch/wizardflow/CheckPendingAgent.js

```diff
@@ -73,6 +74,10 @@ foam.CLASS({
         this.cancelled = true;
         this.sequence.endSequence();
       }
+      if ( ! shouldReopen && statusPending ) {
```

Comment:
It looks like the previous overall condition for ending the sequence was the following:

```
( status !== AVAILABLE && ! shouldReopen ) || ( shouldReopen && capabilities.length < 1 )
```

However here, when `showToast=false`, the condition is now this:
```
! shouldReopen && status == PENDING
```

I think a simpler condition is better so this is good, but I also wonder - why not just ` ! shouldReopen `? @Nauna might remember if there's a reason we had it more complicated to begin with.

Reply:
In early days - we had limited cases and may have overlooked the general scope. Think this is just legacy of building what we needed at the time.